tensorrt_llm/_torch/models/modeling_minimaxm3_vl.py

```
@@ -0,0 +1,2092 @@
+            for _ in range(len(images)):
+                content.append({"type": "image"})
+            for _ in range(len(videos)):
+                content.append({"type": "video"})
+            messages = [{"role": "user", "content": content}]
+            try:
+                templated_text = self._tokenizer.apply_chat_template(
+                    messages,
+                    tokenize=False,
+                    add_generation_prompt=True,
+                )
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)